alf/trainers/off_policy_trainer.py

```diff
@@ -85,8 +85,11 @@ def __init__(self, config):
         self._driver_started = False
 
     def _init_driver(self):
-        for _ in range(1, self._config.num_envs):
-            self._create_environment()
+        for i in range(1, self._config.num_envs):
+            # [self._random_seed, self._random_seed + batch_size) has been used
```

Comment:
where is self._random_seed + batch_size used in policy_trainer?

Reply:
in initialize() we first create an environment as the global one

Reply:
Oh, the comment says from random_seed to random_seed + batch have been used?  Misunderstood it.